qiskit_experiments/library/characterization/cr_phase.py

```diff
@@ -0,0 +1,128 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Cross resonance Hamiltonian phase characterization."""
+
+from typing import Iterable, List, Optional, Tuple
+import numpy as np
+
+from qiskit.circuit import QuantumCircuit, Parameter, Gate
+from qiskit.providers import Backend
+from qiskit.pulse import ScheduleBlock
+
+from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.curve_analysis import OscillationAnalysis
+
+
+class CrossResonancePhase(BaseExperiment):
+    """An experiment to characterize phase errors in cross-resonance gates.
+
+    # section: overview
+
+        This experiment scans the phase of the cross-resonance drive to find the
+        value that most closely produces a ZX-like drive. The following circuits
+        are executed
+
+        .. parsed-literal::
+
+                 ┌───┐┌─────────┐┌───┐┌─────────┐
+            q_0: ┤ X ├┤0        ├┤ X ├┤0        ├───
+                 └───┘│  crp(φ) │└───┘│  crm(φ) │┌─┐
+            q_1: ─────┤1        ├─────┤1        ├┤M├
+                      └─────────┘     └─────────┘└╥┘
+            c: 1/═════════════════════════════════╩═
+                                                  0
+
+        Here, the phase φ is scanned over a range. The resulting oscillation is fit
+        to a cosine function.
+
+    """
+
+    def __init__(
+        self,
+        qubits: Tuple[int, int],
+        crp: ScheduleBlock,
```

Comment:
Why we cannot pass `szx` gate as done in fine amp? A chain of calibrations with different API seems confusing.